ground-server/src/app/controls/page.tsx

```diff
@@ -10,96 +10,129 @@ export default async function Controls() {
-          offButtonLabel="Deactivate"
-          offStateLabel="Deactivated"
-          offCommand={{
-            ignite: false,
-          }}
-          initialStateLabel="No Data"
-        />
+          <ActuationBox
+            title="Igniter"
+            buttons={[
+              {
+                label: "Activate",
+                stateLabel: "Activated",
+                command: { ignite: true },
+                isOn: true
+              },
+            ]}
+            initialStateLabel="No Data"
+          />
 
-        <ActuationBox
-          title="Ball Valve Open/Closed"
-          onButtonLabel="Open"
-          onStateLabel="Opened"
-          onCommand={{
-            bv1Open: true,
-          }}
-          offButtonLabel="Close"
-          offStateLabel="Closed"
-          offCommand={{
-            bv1Open: false,
-          }}
-          initialStateLabel="No Data"
-        />
+          <ActuationBox
+            title="Solenoid Valve 1"
+            buttons={[
+              {
+                label: "Open",
+                stateLabel: "Opened",
+                command: { sv1Open: true },
+                isOn: true
+              },
+              {
+                label: "Close",
+                stateLabel: "Closed",
+                command: { sv1Open: false },
+                isOn: false
+              },
+            ]}
+            initialStateLabel="No Data"
+          />
 
-        <ActuationBox
-          title="Ball Valve On/Off"
-          onButtonLabel="On"
-          onStateLabel="On"
-          onCommand={{
-            bv1Off: false,
-          }}
-          offButtonLabel="Off"
-          offStateLabel="Off"
-          offCommand={{
-            bv1Off: true,
-          }}
-          initialStateLabel="No Data"
-        />
+          <ActuationBox
+            title="Ball Valve"
+            buttons={[
+              {
+                label: "Open",
+                stateLabel: "Opened",
+                command: { bv1Open: true },
+                isOn: true,
+              },
+              {
+                label: "Close",
+                stateLabel: "Closed",
+                command: { bv1Open: false },
+                isOn: false,
+              },
+            ]}
+            initialStateLabel="No Data"
+            useSwitch={false}
+            switchLabel="Ball Valve On/Off"
+            switchOnCommand={{ bv1Off: false }}
+            switchOffCommand={{ bv1Off: true }}
+          />
 
-        <ActuationBox
-          title="Solenoid Valve 1"
-          onButtonLabel="Open"
-          onStateLabel="Opened"
-          onCommand={{
-            sv1Open: true,
-          }}
-          offButtonLabel="Close"
-          offStateLabel="Closed"
-          offCommand={{
-            sv1Open: false,
-          }}
-          initialStateLabel="No Data"
-        />
 
+        </div>
+      </div>
+
+      <div>
+        <h1 className="text-2xl font-bold mb-1 text-center">Rocket Commands</h1>
```

Comment:
Could we add a rocket command for clearing the SD card?
This would also require adding a new command in the protobuf definition